uptime_kuma_api/api.py

```diff
@@ -500,11 +500,14 @@ def _get_event_data(self, event) -> Any:
         return deepcopy(self._event_data[event].copy())
 
     def _call(self, event, data=None) -> Any:
-        r = self.sio.call(event, data, timeout=self.timeout)
-        if isinstance(r, dict) and "ok" in r:
-            if not r["ok"]:
-                raise UptimeKumaException(r.get("msg"))
+        try:
+            r = self.sio.call(event, data, timeout=self.timeout)
```

Comment:
The response of `self.sio.call` is not always a dict (for example `api.need_setup`). In this case `r.pop("ok")` fails. You have deleted the code that checks if the response type is a dict.

Also you have deleted the code that checks the value of the `r["ok"]` value. The UptimeKumaException is now no longer raised when Uptime Kuma responds with an error message.